stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -717,21 +717,23 @@ function exp!(A::StridedMatrix{T}) where T<:BlasFloat
         end
         for k in 2:(div(length(C), 2) - 1)
             P *= A2
-            for ind in eachindex(P)
+            for ind in eachindex(P, U, V)
                 U[ind] += C[2k + 2] * P[ind]
                 V[ind] += C[2k + 1] * P[ind]
             end
         end
 
-        U = A * U
+        U′ = P # alias for the temporary matrix
+        mul!(U′, A, U)
```

Comment:
Why can't you simply continue to use `P` here?

Reply:
Certainly possible. I just thought that using a name similar to `U`  might make the following code easier to follow. But I'm happy to use `P` here as well.

Reply:
Obviously, not super relevant, but I got confused for a moment by the superscript dash. You could perhaps keep the comment (perhaps specifying that you reuse `P` for `A*U`, even though that's obvious from the very next line) and switch to `P`?

Reply:
I've updated the code to use `P` now